pokemon-showdown

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env node
 'use strict';
 
+const spawnSync = require('child_process').spawnSync;
```

Comment:
I'm fine with explicitly importing modules, but not individual functions from modules. Import `child_process` and I'll only be mildly bothered by the lack of camelcase.